components/infra-proxy-service/pipeline/phase1_migration/phase1_migration.go

```diff
@@ -0,0 +1,186 @@
+package phase1_migration
+
+import (
+	"context"
+	"fmt"
+)
+
+type Result struct {
+	Meta         *Meta
+	ParsedResult *ParsedResult
+	Context      context.Context
+}
+
+type Meta struct {
+	StageResults []StageResult
+}
+
+type StageResult struct {
+	StageName string
+	IsSuccess bool
+	Failure   error
+}
+
+type ParsedResult struct {
+	Orgs      []Org
+	Users     []User
+	OrgsUsers []OrgsUsersAssociations
+	KeyDump   KeyDump
+}
+
+// type OrgsUsersAssociations map[User][]Org
+type OrgsUsersAssociations struct {
+	OrgName Org
+	Users   []User
```

Comment:
Instead of using User here we can create a minified structure:
UserAssociation: 
```
{
    Username
    IsAdmin
}
```